### PR TITLE
add styling

### PR DESCRIPTION
**Pull request by milodevhh**
Hi, please have a look over the first Styling...

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  text-transform: uppercase;
+}
+
+header {
+  background: #030007;
+  color: #ffffff;
+  height: 100px;
+  width: 100%;
+  position: fixed;
+  margin: 0 0 0 0;
```
Comment (ninaassmann) [inline css/styles.css]: kurzform:

margin 0;

allerdings würde ich empfehlen ein reset oben bei dem * selektor zu machen

* {
margin: 0;
padding: 0;

**Review by ninaassmann — CHANGES_REQUESTED** [review]
good start

File: bookmarks.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="/css/styles.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Quiz App</h1>
+    </header>
+    <!-- I will use question-card as a class later to style it - What is better to use - or _ when I write a class?  -->
```
Comment (julialoeschel) [inline bookmarks.html:13]: I guess we dont need this anymore :) ?

File: bookmarks.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    <section class="question-card">
+      <div class="div_question-card">
+        <h2>What is Riesling?</h2>
+        <button aria-label="Klick here to get answere">Show Answer</button>
+        <p>A Grape variety, that was esay</p>
+        <button>Hide Answer</button>
+      </div>
+      <!-- I will use categories as a class to style it later -->
```
Comment (julialoeschel) [inline bookmarks.html:21]: this as well :)

File: bookmarks.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      <ul class="question-card_categories">
+        <li>#wine</li>
+        <li>#basic</li>
+        <li>#beverage</li>
+      </ul>
+      <!-- I have to add a Element for a bookmark as well, simple as a <P> Element? -->
```
Comment (julialoeschel) [inline bookmarks.html:27]: 👁️

File: css/styles.css [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+.div_question-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.question-card_categories {
+  display: flex;
+  list-style-type: none;
+  position: absolute bottom -5px left -5px;
```
Comment (julialoeschel) [inline css/styles.css]:
```suggestion
  position: absolute; 
  bottom: -5px ;
  left: -5px;
```

there is no shorthand for this.

File: index.html [hunk 6]
```diff
@@ -4,33 +4,67 @@
...
     <header>
       <h1>Quiz App</h1>
     </header>
     <!-- I will use question-card as a class later to style it - What is better to use - or _ when I write a class?  -->
     <section class="question-card">
-      <h2>What is Riesling?</h2>
-      <button aria-label="Klick here to get answere">Show Answer</button>
-      <p>A Grape variety, that was esay</p>
-      <button>Hide Answer</button>
+      <div class="div_question-card">
```
Comment (julialoeschel) [inline index.html:15]: a card is a perfect example to use an `article` or a `section`.
its more sematic than a `div`

File: index.html [hunk 7]
```diff
@@ -4,33 +4,67 @@
...
+      </section>
     <footer>
-      <nav aria-labelledby="">
-        <ul>
-          <li>Quiz</li>
-          <li>Bookmarks</li>
-          <li>Profile Page</li>
+      <nav>
+        <ul class="nav">
+          <li><a class="nav_link" href="index.html">Quiz</a></li>
```
Comment (julialoeschel) [inline index.html:65]: these links are not going anywhere :)

**Review by julialoeschel — COMMENTED** [review]
lgtm 🚀 
good job!

File: css/styles.css (hunk 1 above)
Comment (milodevhh) [reply css/styles.css]: When I set the rest the class of the question card will flip. May We can look into it together?

**Pull request merged**